### ISSUE TITLE
Startup & argument handling needs an overhaul

### ISSUE THREAD

**Issue by krestenkrab** [trifork/erjang#68]
In many cases, code that run fine when run using "plain ej" with a bunch of `-pa` arguments doesn't work when subjected to the intricate arguments used in an erlang release.
- the `-args_file` option needs to "expand" the file's contents before argument processing
- When running via a release (as opposed to the installed OTP_HOME), erjang can't find certain modules at startup (often `error_handler`). 

In short, we need to do an overhaul of the launch sequence.

This should be a relatively easy project to get acquainted with the Erjang codebase, most notably because it does not involve the JIT compiler, and should be debuggable before anything tricky happens in the VM.
